pyproject.toml

```diff
@@ -122,6 +122,8 @@ core = [
 	"platformdirs >= 2.6.2",
 ]
 
+[tool.setuptools_scm]
```

Comment:
I just noticed that this line gets moved, but it was already present. I'll move it back.